sklearn/manifold/t_sne.py

```diff
@@ -808,17 +812,27 @@ def _tsne(self, P, degrees_of_freedom, n_samples, random_state,
         # * early exaggeration with momentum 0.5
         # * early exaggeration with momentum 0.8
         # * final optimization with momentum 0.8
+
+        # TODO sklearn 0.20: remove this warning
+        if self.learning_rate == "200 (new default)":
+            import warnings
+            warnings.warn("The default learning rate of TSNE has changed " +
+                          "from 1000 to 200 in sklearn 0.19. Set the " +
+                          "learning rate explicitely to avoid this message.")
```

Comment:
Maybe we could add something saying that doing nothing and waiting for 0.20 to be released is an option.